scripts/usdm/process_usdm.py

```
@@ -60,8 +61,7 @@ def workflow(dt: date, routes):
     zipfp = zipfile.ZipFile(tmp.name, "r")
     shpfn = None
     for name in zipfp.namelist():
-        with open(f"/tmp/{name}", "wb") as fp:
-            fp.write(zipfp.read(name))
+        pathlib.Path(f"/tmp/{name}").write_bytes(zipfp.read(name))
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp/"

(S108)